### ISSUE TITLE
How to reach tty1 on live iso?

### ISSUE THREAD

**Issue by Chrysostomus** [manjaro/manjaro-tools-iso-profiles#131]
On my builds the boot process never reaches getty. After boot messages stop I can switch to tt{2-7}, but tty1 never comes.

Usually the last message I get is "Reached target Graphical Interface". Could this be part of the problem? Bspwm edition uses just getty as a login manager.

**Comment by udeved** [comment]
systemd funnily uses tty1, whereas openrc versions used standard tty7.
You just have to switch to a tty2+ o systemd

**Comment by Chrysostomus** [comment]
Okay, let me rephrase the question: how can I reach a tty without user intervention at boot? The problem is that user is not taken to the login prompt from the boot messages. The number of tty is not essential.

**Comment by udeved** [comment]
No DM right?
I have no diea with systemd stuff.
Perhaps change systemd default VT?

**Comment by Chrysostomus** [comment]
Ok, thanks for the lead. Yep, no dm. I wonder if same thing happens with net edition. If not, then I'll just have to find out the difference...

**Comment by udeved** [comment]
Post output of buildiso -p bspwm -qv

Perhpas also build with log enabled.
Whats the build output saying after the custom image packages install?

**Comment by udeved** [comment]
It should say: "No DE detected"

**Comment by Chrysostomus** [comment]
Here is the output of buildiso -p -qv:
```
==> manjaro-tools
  -> version: 0.11.3
  -> manjaro_tools_conf: /etc/manjaro-tools/manjaro-tools.conf
==> PROFILE:
  -> sets_dir_iso: /etc/manjaro-tools/iso.d
  -> buildsets: community|default|minimal|netrunner|official|sonar
  -> buildset_iso: bspwm
  -> is_buildset: false
  -> run_dir: /home/matti/manjaro-tools-iso-profiles
==> OPTIONS:
  -> arch: x86_64
  -> branch: stable
  -> chroots_iso: /media/BUILD/build/buildiso
  -> initsys: systemd
  -> kernel: linux44
==> ARGS:
  -> clean_first: true
  -> images_only: false
  -> iso_only: false
  -> sign: false
  -> is_log: false
==> DIST SETTINGS:
  -> dist_name: Manjaro
  -> dist_release: 16.03
  -> dist_codename: Capella
==> ISO SETTINGS:
  -> iso_label: MJRO1603
  -> iso_publisher: Manjaro Linux <http://www.manjaro.org>
  -> iso_app_id: Manjaro Linux Live/Rescue CD
  -> iso_compression: xz
  -> iso_checksum: md5
  -> use_overlayfs: true
  -> iso_fs: overlayfs
==> BUILD QUEUE:
 --> Profile: [bspwm]
  -> work_dir: /media/BUILD/build/buildiso/bspwm/x86_64
  -> iso_dir: /var/cache/manjaro-tools/iso/community/bspwm/16.03/x86_64
  -> iso_file: manjaro-bspwm-community-16.03-x86_64.iso
  -> displaymanager: none
  -> autologin: false
  -> nonfree_xorg: true
  -> pxe_boot: false
  -> plymouth_boot: false
  -> multilib: false
  -> efi_boot_loader: grub
  -> efi_part_size: 31M
  -> hostname: manjaro
  -> username: manjaro
  -> password: manjaro
  -> addgroups: video,audio,power,disk,storage,optical,network,lp,scanner,wheel
  -> start_systemd: bluetooth cronie ModemManager NetworkManager org.cups.cupsd tlp tlp-sleep
  -> start_systemd_live: manjaro-live mhwd-live pacman-init
  -> disable_systemd: pacman-init
  -> is_custom_pac_conf: true

```

I don't remember getting that message. I'll try rebuilding. How can I get buildiso to generate a log? My /var/log/manjaro-tools is empty.

**Comment by udeved** [comment]
-l option attached

**Comment by Chrysostomus** [comment]
Thanks

**Comment by udeved** [comment]
Btw, if you place a copy of manjaro-tools.conf in $HOME/.config/manjaro-tools, its much easier to set stuff without root permissions.

You can also copy over the sets dirs there.

**Comment by Chrysostomus** [comment]
Okay, on the newest rebuild I got the message
` -> Detected desktop environment: none
`
However, I also update manjaro-tools before this build. I'll see if this time works different.

**Comment by udeved** [comment]
can you look in chroot if a displaymanager symlink is created in image chroot /etc/systemd/system?
It should not be there.

**Comment by Chrysostomus** [comment]
It is not.

**Comment by Chrysostomus** [comment]
The iso is now compressing. I'll see how it boots before going to sleep.

**Comment by Chrysostomus** [comment]
Okay, issue persists (along with the other weird issue I have https://forum.manjaro.org/index.php?topic=31744.0 . But that is off topic). I get "Reached target Graphical Interface" and that is all.

**Comment by Chrysostomus** [comment]
Now getting somewhere...
getty service is not running. By default, systemd runs 1 getty, and starts more as you move to new ttys. But now th inital one is not getting ran. I'll try adding getty@tt1.service to profile.conf. Tomorrow. It already too late and I have work tomorrow...

**Comment by Chrysostomus** [comment]
New data:
Having getty@tty1.service enabled fixes the issue. However, adding getty@tty1 does not enable it. I manually copied the files to chroot, then it work.

**Comment by Chrysostomus** [comment]
Adding getty@ to profile.conf works. Thanks for help!
